tests/StacktraceTest.php

```diff
@@ -56,6 +58,7 @@ public function testStacktraceJsonSerialization(): void
 
         $stacktrace->addFrame('path/to/file', 1, ['file' => 'path/to/file', 'line' => 1, 'function' => 'test_function']);
         $stacktrace->addFrame('path/to/file', 2, ['file' => 'path/to/file', 'line' => 2, 'function' => 'test_function', 'class' => 'TestClass']);
+        $stacktrace->addFrame('path/to/file', 3, ['file' => 'path/to/file', 'line' => 2, 'class' => 'CrashyClass']);
```

Comment:
To be coherent with the rest of the code please use `TestClass` as class name

Reply:
Renamed.